static/js/sefaria/sefaria.js

```diff
@@ -469,31 +469,64 @@ Sefaria = extend(Sefaria, {
         });
   },
   _bulkTexts: {},
+  partitionArrayForURL: function(urlMaxLength, arr, dividerToken) {
```

Comment:
I would switch the positional arguments here arr and urlMaxLength, its more customary for the main input to be the first param.